e2e/src/classes/browser-context.class.ts

```diff
@@ -2,30 +2,36 @@ import { Browser, Page } from 'puppeteer';
 
 import { getEnv } from '../utils/env.utils';
 
-const defaultSeedPhrase = getEnv('DEFAULT_SEED_PHRASE');
-const defaultPassword = getEnv('DEFAULT_PASSWORD');
-const defaultPrivateKey = getEnv('DEFAULT_HD_ACCOUNT_PRIVATE_KEY');
-const secondSeedPhrase = getEnv('SECOND_SEED_PHRASE');
-const privateKeyOfSecondSeedPhrase = getEnv('SECOND_SEED_PHRASE_PRIVATE_KEY');
+const DEFAULT_HD_ACCOUNT_SEED_PHRASE = getEnv('DEFAULT_HD_ACCOUNT_SEED_PHRASE');
+const DEFAULT_HD_ACCOUNT_FIRST_PRIVATE_KEY = getEnv('DEFAULT_HD_ACCOUNT_FIRST_PRIVATE_KEY');
+const DEFAULT_HD_ACCOUNT_SECOND_PRIVATE_KEY = getEnv('DEFAULT_HD_ACCOUNT_SECOND_PRIVATE_KEY');
+const DEFAULT_PASSWORD = getEnv('DEFAULT_PASSWORD');
+const IMPORTED_HD_ACCOUNT_SEED_PHRASE = getEnv('IMPORTED_HD_ACCOUNT_SEED_PHRASE');
+const IMPORTED_HD_ACCOUNT_FIRST_PRIVATE_KEY = getEnv('IMPORTED_HD_ACCOUNT_FIRST_PRIVATE_KEY');
 
-if (!defaultSeedPhrase) throw new Error('process.env.DEFAULT_SEED_PHRASE not found.');
-if (!defaultPassword) throw new Error('process.env.DEFAULT_PASSWORD not found.');
-if (!defaultPrivateKey) throw new Error('process.env.DEFAULT_PASSWORD not found.');
-if (!secondSeedPhrase) throw new Error('process.env.SECOND_SEED_PHRASE not found.');
-if (!privateKeyOfSecondSeedPhrase) throw new Error('process.env.SECOND_SEED_PHRASE_PRIVATE_KEY not found.');
+if (!DEFAULT_HD_ACCOUNT_SEED_PHRASE) throw new Error('process.env.DEFAULT_HD_ACCOUNT_SEED_PHRASE not found.');
+if (!DEFAULT_PASSWORD) throw new Error('process.env.DEFAULT_PASSWORD not found.');
+if (!DEFAULT_HD_ACCOUNT_FIRST_PRIVATE_KEY)
+  throw new Error('process.env.DEFAULT_HD_ACCOUNT_FIRST_PRIVATE_KEY not found.');
+if (!DEFAULT_HD_ACCOUNT_SECOND_PRIVATE_KEY)
+  throw new Error('process.env.DEFAULT_HD_ACCOUNT_SECOND_PRIVATE_KEY not found.');
+if (!IMPORTED_HD_ACCOUNT_SEED_PHRASE) throw new Error('process.env.IMPORTED_HD_ACCOUNT_SEED_PHRASE not found.');
+if (!IMPORTED_HD_ACCOUNT_FIRST_PRIVATE_KEY)
+  throw new Error('process.env.IMPORTED_HD_ACCOUNT_FIRST_PRIVATE_KEY not found.');
```

Comment:
a lot of the same code, refactor this pls

Reply:
refactored to one condition